docs/user-guide/concepts/data-structures.md

```diff
@@ -1,6 +1,6 @@
 # Data structures
 
-The core base data structures provided by Polars are `Series` and `DataFrames`.
+The core base data structures provided by Polars are `Series` and `DataFrame`.
```

Comment:
`LazyFrame` could also be added. Thoughts?

Reply:
Well, technically LazyFrame isn't really a data structure in its own right - it's a query plan _against_ a data structure (specifically, against a DataFrame) ;)

Reply:
Yes, true, good points. Will leave as is.